types/trusted-types/index.d.ts

```diff
@@ -70,7 +62,6 @@ declare global {
         TrustedHTML: TrustedHTML;
```

Comment:
could you delete the uppercase `TrustedTypes` ?

Reply:
Done.